File: Java/Mahasiswa.java
//class mahasiswa meneruskan attribut manusia
public class Mahasiswa extends Human
{
	private String nim;
	private String prodi;
	private String fakultas;

	Mahasiswa() {
		// konstruktor kosong
	}

	Mahasiswa(String nim, String prodi, String fakultas) {
		// konstruktor langsung isi atribut
		this.nim = nim;
		this.prodi = prodi;
		this.fakultas = fakultas;
	}

	//setter and getter//

	public void setNim(String nim) {
		// mengeset nilai nim Mahasiswa
		this.nim = nim;
	}

	public String getNim() {
		//mengembalikan nilai nim mahasiswa
		return this.nim;
	}

	public void setProdi(String prodi) {
		// mengeset nilai prodi Mahasiswa
		this.prodi = prodi;
	}

	public String getProdi() {
		//mengembalikkan nilai prodi mahasiswa
		return this.prodi;
	}

	public void setFakultas(String fakultas) {
		// mengeset nilai fakultas Mahasiswa
		this.fakultas = fakultas;
	}

	public String getFakultas() {
		//mengembalikkan nilai fakultas mahasiswa
		return this.fakultas;
	}

}

File: python/Mahasiswa.py
from Human import Human

class Mahasiswa(Human):

	#private attribute Manusia
	__nim = ""
	__prodi = ""
	__fakultas =""


	def __init__(self, nim="", prodi="", fakultas=""):
		self.__nim = nim
		self.__prodi = prodi
		self.__fakultas = fakultas
	


	#setter and getter#

	def setProdi(self, prodi):
		# mengeset nilai prodi Mahasiswa
		self.__prodi = prodi
	

	def getProdi(self):
		#mengembalikan nilai prodi mahasiswa
		return self.__prodi
	

	def setNim(self, nim):
		# mengeset nilai nim Mahasiswa
		self.__nim = nim
	

	def getNim(self):
		#mengembalikan nilai nim mahasiswa
		return self.__nim
	

	def setFakultas(self, fakultas):
		# mengeset nilai fakultas Mahasiswa
		self.__fakultas = fakultas
	

	def getFakultas(self):
		#mengembalikkan nilai fakultas mahasiswa
		return self.__fakultas
	
	
